docs/sources/developer/architecture/components/api.md

```diff
@@ -0,0 +1,78 @@
+# API
+
+The Faro API is the primary interface for end-users to interact with Faro. It is the central place where signals are
+sent to but it also contains helpers methods for other components like metas.
```

Comment:
nit: a couple of minor suggestions:
> It is the central place where signals are sent [~to~], but it also contains helper[~s~] methods for other components like metas.
-'where signals are sent' (no need for 'sent to', 'to' is implied.)
-'helper methods' (was 'helpers' methods, pluralisation for something like 'helper methods' happens once at the end. You can have 'helpers').

How about:
```
It is the central collector of signals, and also exposes helper methods for extensions such as metas.
```
?